### PR TITLE
[BRC] Implement Terisiare's Devastation

### PR DESCRIPTION
**Pull request by PurpleCrowbar**

### CODE REVIEW

**Comment by PurpleCrowbar** [comment]
[[Terisiare's Devastation]]

File: Mage.Sets/src/mage/cards/t/TerisiaresDevastation.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    private static final DynamicValue xValue = new PermanentsOnBattlefieldCount(new FilterControlledArtifactPermanent(), -1);
+
+    public TerisiaresDevastation(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.SORCERY}, "{X}{2}{B}{B}");
+
+        // You lose X life and create X tapped Powerstone tokens. Then all creatures get -1/-1 until end of turn for each artifact you control.
+        this.getSpellAbility().addEffect(new LoseLifeSourceControllerEffect(ManacostVariableValue.REGULAR));
+        this.getSpellAbility().addEffect(new CreateTokenEffect(new PowerstoneToken(), ManacostVariableValue.REGULAR, true, false).concatBy("and"));
+        this.getSpellAbility().addEffect(new BoostAllEffect(
+                xValue, xValue, Duration.EndOfTurn)
```
Comment (Alex-Vasile) [inline Mage.Sets/src/mage/cards/t/TerisiaresDevastation.java:32]: These should be -xValue, -xValue

Comment (PurpleCrowbar) [reply Mage.Sets/src/mage/cards/t/TerisiaresDevastation.java:32]: The multiplier on line 23 inverts the sign by multiplying by -1, so xValue is a negative number

Comment (Alex-Vasile) [reply Mage.Sets/src/mage/cards/t/TerisiaresDevastation.java:32]: You're right. I missed that.

**Pull request merged**